src/java/org/apache/cassandra/tcm/RemoteProcessor.java

```diff
@@ -152,15 +152,21 @@ public ClusterMetadata fetchLogAndWait(Epoch waitFor, Retry.Deadline retryPolicy
     }
 
     @Override
-    public LogState reconstruct(Epoch lowEpoch, Epoch highEpoch, Retry.Deadline retryPolicy)
+    public LogState getLocalState(Epoch start, Epoch end, boolean includeSnapshot, Retry.Deadline retryPolicy)
+    {
+        return log.getLocalEntries(start, end, includeSnapshot);
+    }
+
+    @Override
+    public LogState getLogState(Epoch lowEpoch, Epoch highEpoch, boolean includeSnapshot, Retry.Deadline retryPolicy)
```

Comment:
why does the retry policy only reflect the `request.get` timeout and isn't used for the networking retries?  all the attempt logic is ignored and just the upper limit is leveraged, so we don't have `retry` respected?

Reply:
You are right; passed `retryPolicy` to `sendWithCallbackAsync`.